src/include/sof/lib/dma.h

```diff
@@ -149,6 +149,8 @@ struct dma_chan_status {
 	uint32_t w_pos;
 	uint32_t r_pos;
 	uint32_t timestamp;
+
+	void *private;
```

Comment:
Can you add a comment here describing this is for ipc dependent position info. I would rename to ```ipc_posn_data```.

Reply:
sure